src/actions/editor.js

```diff
@@ -420,6 +400,49 @@ export function sendRecurringData(formValues, contentLanguages, user, updateExis
     }
 }
 
+export function updateRecurringData(superEvent, subEventsToUpdate, contentLanguages, user, superEventId) {
+    return (dispatch) => {
+        const keysNotToUpdate = ['start_time', 'end_time', 'id', 'super_event_type']
+        // update sub events based on super event data
+        const updatedSubEvents = subEventsToUpdate.map(subEvent => {
+            const updatedSubEvent = Object.keys(subEvent)
+                // filter out keys that we don't want to update
+                .filter(key => !keysNotToUpdate.includes(key))
+                .reduce((acc, key) => {
+                    if (!isNil(superEvent[key])) {
+                        acc[key] = superEvent[key]
+                    }
+                    return acc
+                }, subEvent)
+
+            return {...updatedSubEvent, ...nullifyMultiLanguageValues(updatedSubEvent, contentLanguages)}
+        })
+
+        // prepare other needed information for request metadata
```

Comment:
Hm. I would rather refactor (if possible) this method to use the same `dispatch(executeSendRequest)` method that is used in all other POST and PUT cases.

Certainly that method will need some changes, i.e. checking that if `updateExisting` is True but still we are PUTting several events, then it would behave differently.

Still, I think it is best to have handling all the event POST/PUT functionality with the same method, as it already does stuff as error handling and handling creation of multiple events. It would be weird to handle updating multiple events differently.